### DISCUSSION TITLE
about updating engine version

### DISCUSSION THREAD

**Discussion by GV123h** [ikemen-engine/Ikemen-GO#1255] [Q&A]
Is my sace>config.json has many custom configuration and defined ssz,

Is it possible to update an ikemen build by simply coping the updated .exe to the ikemen root folder replacing the old one?

thanks in advance for your comments

**Comment by Lazin3ss** [comment]
To update an Ikemen version, you have to copy the next files and folders:
- Whole "Data" folder
- Whole "External" folder
- Whole "Font" folder
- Ikemen GO executable file

And that's it. Everything else is irrelevant.

**Comment by GV123h** [comment]
Hi

Copy these contents from the updated build to my build?

Or the other way around ?

De: Wreq! ***@***.***>
Enviado el: lunes 29 de mayo de 2023 10:06
Para: ikemen-engine/Ikemen-GO ***@***.***>
CC: GV123h ***@***.***>; Author ***@***.***>
Asunto: Re: [ikemen-engine/Ikemen-GO] about updating engine version (Discussion #1255)


To update an Ikemen version, you have to copy the next files and folders:

  *   Whole "Data" folder
  *   Whole "External" folder
  *   Whole "Font" folder
  *   Ikemen GO executable file

And that's it. Everything else is irrelevant.

-

**Comment by potsmugen** [comment]
From the updated build to your build. But make a backup before you do. Especially for select.def.

You may also need to delete config.json so Ikemen makes a new one.
